orca-pipelinetemplate/src/main/java/com/netflix/spinnaker/orca/pipelinetemplate/v1schema/render/YamlRenderedValueConverter.java

```diff
@@ -24,6 +26,8 @@
 
 public class YamlRenderedValueConverter implements RenderedValueConverter {
 
+  private static final List<String> yamlKeywords = Arrays.asList("yes", "no", "on", "off");
```

Comment:
`YAML_KEYWORKS` would be more in keeping with naming conventions for `static final` fields.